### ISSUE TITLE
Content: cloud.gov intro section

### ISSUE THREAD

**Issue by coreycaitlin** [18F/18f.gsa.gov#2068]
We need content for the project-tag landing page about cloud.gov.

This is issue is done when:

- [x] Content is drafted
- [x] Content is reviewed and revised with outreach team
- [x] Content is reviewed by cloud.gov team

**Comment by coreycaitlin** [comment]
Pushing this out a sprint, in favor of starting with a few agency-centric stories.

**Comment by coreycaitlin** [comment]
Source content from old intake site:

> cloud.gov includes a Platform-as-a-Service (PaaS) specifically built for government, based on the open source Cloud Foundry. It provides a secure, scalable platform for teams to build and host their application or website. It can help development teams work faster and cheaper by handling much of their deployment complexity. The cloud.gov platform is currently in a small pilot program, but we will be expanding access over time.

**Comment by awfrancisco** [comment]
We should have new content to help feed this soon, and it'd be good to move this up in line if we can.

**Comment by coreycaitlin** [comment]
James and Britta have been the content designers on this project.

**Comment by acolter** [comment]
Draft goes here: https://docs.google.com/document/d/1nnQhBCgYwErg0EJh99hKqgjjdNvDnm__XB3GgwsCkfI/edit

**Comment by acolter** [comment]
The cloud.gov folks have put together a list of customers here: https://docs.google.com/document/d/1PEzMhZIndUIt2GibPYgya2WG3GWV2SjjAh33XmUsE0U/edit Need their permission before including them in the project intro page.

**Comment by awfrancisco** [comment]
Left my edits. Added in a bit on the FedRAMP Authorization because I think it's important to include, but open to the idea of moving that bit or rewording it.

**Comment by acolter** [comment]
Rewrote the cloud.gov project intro based on feedback from Britta. Once this team has reviewed the rewrite, the clould.gov team should have a chance to review it again.

**Comment by coreycaitlin** [comment]
This is great! Thank you so much for working through this one.

**Comment by acolter** [comment]
Alright, I admit my rewrite of this project intro was a steaming pile of poo. Probably because at a basic level I don't really understand what cloud.gov is or how to explain it in plain English to a lay audience. Anyway, I gave it another go. @coreycaitlin let me know if it's any better now.

**Comment by awfrancisco** [comment]
This looks great! I didn't detect anything steaming.

**Comment by awfrancisco** [comment]
@acolter Is the next step for this to be reviewed by the cloud.gov team?

**Comment by acolter** [comment]
Yes, they should look at the rewrite.

**Comment by awfrancisco** [comment]
@elainekamlley this is also ready for production

**Comment by awfrancisco** [comment]
https://18f.gsa.gov/what-we-deliver/cloud-gov/
